### PR TITLE
modified endpoint integrated with watson

### PR DESCRIPTION
**Pull request by VaishnaviHire**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -77,13 +70,14 @@ def create_sessions():
         return Response(json.dumps(status), status=400, mimetype='application/json')
 
 
-@app.route('/newapp', methods=['POST'])
+@app.route('/newapps', methods=['POST'])
 def handle_new_app():
     if request.method == 'POST':
         try:
-            app = request.get_json()['apps'][0]
+            verify_request(request.headers.get('App_Bot_Api_Secret'))
```
Comment (Noah-Huppert) [inline app.py]: The severless registry API puts the secret in the `Authorization` header, I can change that though.

Comment (Noah-Huppert) [reply app.py]: I'll put the secret wherever, if you could document where I should put it in [DESIGN.md](https://github.com/kscout/chat-bot-api/blob/master/DESIGN.md#new-app-upload) I'll update the serverless-registry-api accordingly

Comment (VaishnaviHire) [reply app.py]: Its better to put that in Authorization header. I will update the function and DESGIN.md. You don`t need to make any change

Comment (Noah-Huppert) [reply app.py]: Thank you!

**Pull request merged**